src/Microsoft.DotNet.Wpf/test/PublishAfterBuild.targets

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What are the pros/cons of this target being left here, vs. being moved under eng\WpfArcadeSdk\tools/

Reply:
Likewise, discuss each individualized props/targets file that appears under test\. Each of them breaks the general model that build-related intelligence is refactored out of src\ into a separate location.